plugins/generic/datacite/filter/DataciteXmlFilter.php

```diff
@@ -281,18 +281,8 @@ public function createRootNode($doc)
 
     /**
      * Create creators node.
-     *
-     * @param \DOMDocument $doc
-     * @param \APP\issue\Issue $issue
-     * @param \APP\publication\Publication $publication
-     * @param \PKP\galley\Galley $galley
-     * @param \PKP\submissionFile\SubmissionFile $galleyFile
-     * @param string $publisher
-     * @param array $objectLocalePrecedence
-     *
-     * @return \DOMElement
      */
-    public function createCreatorsNode($doc, $issue, $publication, $galley, $galleyFile, $publisher, $objectLocalePrecedence)
+    public function createCreatorsNode(DOMDocument $doc, Issue $issue, Publication $publication, Galley $galley, SubmissionFile $galleyFile, string $publisher, array $objectLocalePrecedence): DOMNode
```

Comment:
In testing I encountered many cases of ` Argument #4 ($galley) must be of type PKP\galley\Galley, null given` (and same for `SubmissionFile`) - should these be adjusted to take null values as well?